src/store/lib/useControlledPointsStore.js

```diff
@@ -61,6 +61,7 @@ export default function useControlledPointsStore() {
         );
       }
     } catch (error) {
+      console.error(error);
```

Comment:
If we want to leave this in (which is reasonable) we might want to tack on a little message.

```suggestion
      console.error('failed to fetch controlled points', error);
```